pkg/cmd/experimental/syncgroups/ad/ldapinterface.go

```diff
@@ -73,15 +73,17 @@ func (e *ADLDAPInterface) ExtractMembers(ldapGroupUID string) ([]*ldap.Entry, er
 			return nil, err
 		}
 
-		for i := range currEntries {
-			currEntry := currEntries[i]
-
+		for _, currEntry := range currEntries {
 			if !isEntryPresent(usersInGroup, currEntry) {
 				usersInGroup = append(usersInGroup, currEntry)
 			}
 		}
 	}
 
+	if len(usersInGroup) == 0 {
+		return nil, ldaputil.NewEntryNotFoundError()
```

Comment:
I'm not completely happy exposing this error outside of the `ldaputil` package but unless we want to create our own version of it internally, we need this here.